File: PHP/1158.php
<?php
// Problema: 1158 - Soma de Ímpares Consecutivos III | Resposta: Accepted
// Linguagem: PHP (8.1.2) [+1s]                      | Tempo: 0.211S

$rep = intval(fgets(STDIN));
for ($i = 0; $i < $rep; $i++) {
    [$x, $y] = array_map('intval', explode(" ", fgets(STDIN)));

    if ($x % 2 == 0) $x++;

    $soma = $y * ($x + $y - 1);
    echo $soma . PHP_EOL;
}
?>

File: C++/1158.cpp
// Problema: 1158 - Soma de Ímpares Consecutivos III | Resposta: Accepted
// Linguagem: C20++ [+0s]                            | Tempo: 0.000s

#include <iostream>

using namespace std;
int main(){
    int rep, x, y, soma;
    cin >> rep;
    
    while(rep--){
        soma = 0;
        cin >> x >> y;
        if(x%2 == 0) x++;

        while(y--){
            soma += x;
            x += 2;
        }

        cout << soma << endl;
    }
    
    return 0;
}